src/app/components/UserAssets/index.tsx

```diff
@@ -20,8 +20,9 @@ export const UserAssets: React.FC = () => {
   const account = useAccount();
   const { checkMaintenances, States } = useMaintenance();
   const {
-    [States.FASTBTC]: fastBtcLocked,
-    [States.TRANSACK]: transackLocked,
+    [States.FASTBTC_SEND]: fastBtcSendLocked,
+    [States.FASTBTC_RECEIVE]: fastBtcReceiveLocked,
+    [States.TRANSAK]: transackLocked,
```

Comment:
Again, this wasn't introduced in this PR but I noticed that this state doesn't do anything :D We would need to re-implement Transak maintenance state because I suppose it broke when we needed to change the original Transak implementation.

@soulBit what do you want to do about that? Will you create a new task or should it be fixed in this one? You can test it by setting Transak maintenance mode.